hosts_example.yml

```diff
@@ -158,13 +158,29 @@ all:
     #       topics: "test_topic"
 
     #### Configuring Role Based Access Control ####
-    ## To have CP-Ansible configure Components for RBAC and create necessary role bindings, set these Mandtory variables:
-    ## Note: Confluent Components will be configured to connect to the "internal" listener automatically
+    ## To have CP-Ansible configure Components for RBAC and create necessary role bindings, set these mandatory variables:
+    ## Note: Confluent components will be configured to connect to the "internal" listener automatically
     ## DO NOT UPDATE the internal listener
-    ## Note: It is recommended to create an additional listener for external clients, but the interbroker one would also work
+    ## Note: It is recommended to create an additional listener for external clients, but the interbroker listener would also work
     ## Note: An authentication mode must be selected on all listeners, for example (ssl_enabled=false and ssl_mutual_auth_enabled=false) or sasl_protocol=none is not supported.
     # rbac_enabled: true
     ##
+    ####  Configuring Role Based Access Control with a remote MDS ####
+    ## To have CP-Ansible configure Brokers and Components for RBAC with the MDS on a remote cluster, set these mandatory variables:
+    ## Note: Confluent components will be configured to connect to the "internal" listener automatically and to use the URL for the remote cluster for MDS
+    ## DO NOT UPDATE the internal listener
+    ## Note: It is recommended to create an additional listener for external clients, but the interbroker listener would also work
+    ## Note: An authentication mode must be selected on all listeners, for example (ssl_enabled=false and ssl_mutual_auth_enabled=false) or sasl_protocol=none is not supported.
+    # rbac_enabled: true
+    # external_mds_enabled: true
+    # external_super_user: mds
```

Comment:
is this missing mds_ssl_enabled: true/false?

Reply:
It is missing, you're right!  It's also missing in the sample inventory.  Now corrected.  Will push once all changes are complete.